designer/app.py

```diff
@@ -922,15 +912,15 @@ def _fbrowser_load(self, instance):
             return
 
         file_path = instance.selection[0]
-        file_extension = instance.selection[0].split('.')
+        file_name, file_extension = os.path.splitext(instance.selection[0])
 
         error = None
         try:
-            if file_extension[1] == 'py':
+            if file_extension == '.py':
```

Comment:
@manthansharma but since we're just using the project path to open the project, let's accept .py, .kv, and buildozer.spec. It's more flexible :)